src/xo-app/new-vm/index.js

```diff
@@ -922,10 +928,20 @@ export default class NewVm extends BaseComponent {
             onChange={this._getOnChange('coresPerSocket')}
             value={coresPerSocket}
           >
-            <option> Choose core(s) per socket </option>
-            {map(this._getCoresPerSocketPossibilities(),
-              coresPerSocket => <option value={coresPerSocket}>{`${CPUs / coresPerSocket} socket${CPUs / coresPerSocket > 1 ? 's' : ''} with ${coresPerSocket} core${coresPerSocket > 1 ? 's' : ''} per socket`}</option>
-            )}
+            <option>{formatMessage(messages.vmChooseCoresPerSocket)} </option>
```

Comment:
Avoid using `formatMessage()` because it has some limitations, do this instead:

```js
{_('vmChooseCoresPerSocket', message => <option>{message}</option>)}
```